docs/demo_02.md

```diff
@@ -177,7 +175,7 @@ tx4 =
   Transaction.new([{spend_blknum, 0, 0}], [{bob.addr, eth, 7}]) |>
```

Comment:
in line 169
```elixir
~c(echo '{"address": "#{bob_enc}"}' | http POST #{watcher_url}/utxo.get) |>
```
change to:
```elixir
~c(echo '{"address": "#{bob_enc}"}' | http POST #{watcher_url}/account.get_utxos) |>
```

Reply:
If we're on this... is there a particular reason for echo examples? It's a bit odd.

Reply:
To demonstrate our full geekiness I guess. No idea, maybe to better expose sent data ❓

Reply:
hm, that's how httpie likes to have its json served, I think. But not much reason, really, "if it works, it works".